front/src/components/Table/Table.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import styles from './Table.module.css';
+import { useSelector } from 'react-redux';
+import DataRow from '../DataRow/DataRow';
+import HeadRow from '../HeadRow/HeadRow';
+import Footer from '../Footer/Footer';
+
+function Table(props) {
+    const {headers} = props;
+    const columns = Object.assign({}, ...headers.map(head => ({[head.name]: head.flex})));
```

Comment:
Очень неявное преобразование, сделай лучше через reduce
```suggestion
    const columns = headers.reduce((acc, {name, flex}) => {
       acc[name] = flex;
       return acc;
},{});
```

Reply:
Лучше все преобразования сделать в селекторах, а в компонент передать готовые пропсы